src/main/java/uk/gov/justice/hmpps/prison/api/resource/ScheduleResource.java

```diff
@@ -89,9 +99,15 @@ public List<PrisonerSchedule> getLocationEvents(@PathVariable("agencyId") @Param
             @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
             @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Get all Prisoner activities for given date at location.", description = "Get all Prisoner activities for given date at location.")
+    @PreAuthorize("hasRole('NOMIS_ACTIVITIES')")
```

Comment:
All the PreAuthorize's have been added. Role NOMIS_ACTIVITIES for activity / appointment related endpoints

Reply:
I agree, feels like it will confuse matters with sync/migration

Reply:
Oh dear, ok .. unfortunately I already added it to OffenderActivitiesResource in an earlier PR !

Reply:
.. but I will change it as discussed